src/battle_message.c

```diff
@@ -1400,6 +1404,7 @@ const u16 gFirstTurnOfTwoStringIds[] =
     STRINGID_VANISHEDINSTANTLY,         // MOVE_PHANTOM_FORCE
     STRINGID_PKNMABSORBINGPOWER,        // MOVE_GEOMANCY
     STRINGID_CLOAKEDINAFREEZINGLIGHT,   // MOVE_FREEZE_SHOCK
+	STRINGID_PKMNTOOKTARGETHIGH, //MOVE_SKY_DROP
```

Comment:
Replace that tab with 4 empty spaces and add some spaces inbetween the last comma and the comment in order to align it properly with the ones in the lines above it.